proposals/lip-0027.md

```diff
@@ -84,9 +84,9 @@ To solve the above shortcomings of JSON schema validation, we propose to introdu
 
 ### Protobuf Encoding
 
-The encoding specification of this proposal is based on [Protocol Buffers](https://developers.google.com/protocol-buffers), a widely used language-neutral serialization method. The encoding and decoding rules were chosen in such a way that any decoder following the protobuf specification with the correct .proto file decodes binary messages correctly. Hence, third party implementations could use other protobuf implementations to decode binary messages derived from this protocol. [Appendix B](#appendix-b-json-schema-to-protobuf) describes how to create a .proto file corresponding to a Lisk JSON schema.
+The encoding specification of this proposal is based on [Protocol Buffers](https://developers.google.com/protocol-buffers), a widely used language-neutral serialization method. The encoding and decoding rules were chosen in such a way that any decoder following the protobuf specification with the correct .proto file decodes valid binary messages correctly. Hence, third party implementations could use other protobuf implementations to decode binary messages derived from this protocol. [Appendix B](#appendix-b-json-schema-to-protobuf) describes how to create a .proto file corresponding to a Lisk JSON schema.
```

Comment:
```suggestion
The encoding specification of this proposal is based on [Protocol Buffers](https://developers.google.com/protocol-buffers), a widely used language-neutral serialization method. The encoding and decoding rules were chosen in such a way that any decoder following the protobuf specification with the correct `.proto` file decodes valid binary messages correctly. Hence, third party implementations could use other protobuf implementations to decode binary messages derived from this protocol. [Appendix B](#appendix-b-json-schema-to-protobuf) describes how to create a .proto file corresponding to a Lisk JSON schema.
```

Reply:
Or maybe normal quotes, just to distinguish the file name. Notice that there are several occurrences I didn't highlight

Reply:
Change here and in all other occurrences.